src/components/breadcrumb/components/breadcrumb-item/breadcrumb-item.stories.js

```diff
@@ -0,0 +1,35 @@
+import { html } from 'lit-html';
+import { docsIntro, wrapWidth, TYPES } from '../../../../../.storybook/utils.js';
+import '../..';
+
+export default {
+  title: 'custom-elements/vl-breadcrumb/vl-breadcrumb-item',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'breadcrumb',
+          stylesheets: ['breadcrumb'],
+          intro: 'Use breadcrumb items to add extra navigation steps to a breadcrumb.',
+        }),
+      },
+    },
+  },
+  args: {
+    href: '',
```

Comment:
Je kan als value van die href hier default `#` meegeven, dan kan dat stukje logica uit je story, want dat is best verwarrend.